Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/AssetBrowserModel.cpp

```diff
@@ -226,26 +230,38 @@ namespace AzToolsFramework
             Path newPath = oldPath;
             newPath.ReplaceFilename(newFile);
             newPath.ReplaceExtension(extension);
-            using SCCommandBus = AzToolsFramework::SourceControlCommandBus;
-            SCCommandBus::Broadcast(
-                &SCCommandBus::Events::RequestRename, oldPath.c_str(), newPath.c_str(),
-                [&, index, item, newPath](bool success, [[maybe_unused]] const AzToolsFramework::SourceControlFileInfo& info)
+
+            {
+                using namespace AzFramework::AssetSystem;
+                AssetChangeReportRequest moveRequest(
+                    AZ::OSString(oldPath.c_str()), AZ::OSString(newPath.c_str()), AssetChangeReportRequest::ChangeType::Move);
+                AssetChangeReportResponse moveResponse;
+                if (SendRequest(moveRequest, moveResponse))
                 {
-                    if (success)
+                    AZStd::string message;
+                    for (int i = 0; i < moveResponse.m_lines.size(); ++i)
                     {
-                        emit dataChanged(index.parent(), index);
-
-                        if (m_assetEntriesToCreatorBusIds.contains(item))
-                        {
-                            AzToolsFramework::AssetBrowser::AssetBrowserFileCreationNotificationBus::Event(
-                                m_assetEntriesToCreatorBusIds[item],
-                                &AzToolsFramework::AssetBrowser::AssetBrowserFileCreationNotifications::HandleInitialFilenameChange,
-                                newPath.c_str());
-                        }
+                        message += moveResponse.m_lines[i] + "\n";
                     }
 
-                    m_assetEntriesToCreatorBusIds.erase(item);
-                });
+                    if (message.size())
+                    {
+                        QMessageBox msgBox(nullptr);
+                        msgBox.setWindowTitle("After Rename Asset Information");
+                        msgBox.setIcon(QMessageBox::Warning);
+                        msgBox.setText("The asset has been moved.");
+                        msgBox.setInformativeText("More information can be found by pressing \"Show Details...\".");
+                        msgBox.setStandardButtons(QMessageBox::Ok);
+                        msgBox.setDefaultButton(QMessageBox::Ok);
+                        msgBox.setDetailedText(message.c_str());
+                        QSpacerItem* horizontalSpacer2 = new QSpacerItem(600, 0, QSizePolicy::Minimum, QSizePolicy::Expanding);
+                        QGridLayout* moveLayout = (QGridLayout*)msgBox.layout();
+                        moveLayout->addItem(horizontalSpacer2, moveLayout->rowCount(), 0, 1, moveLayout->columnCount());
+                        msgBox.exec();
+                        emit dataChanged(index.parent(), index);
```

Comment:
Is there something special going on with the layout here? Would it be sufficient and easier to use the standard QMessageBox::warning(...) function to prompt instead of manually setting up the UI?

Reply:
Need to adjust the size of the QMessageBox and this is the only way to do it. However, I have replaced this code with a new  function as it is used quite a few times in this code.